components/widgets/users/avatar/avatar.tsx

```diff
@@ -23,11 +27,16 @@ const Avatar = ({
     ...attrs
 }: Props & Attrs) => {
     const classes = classNames(`Avatar Avatar-${size}`, attrs.className);
+    const {centerChannelBg, centerChannelColor} = useSelector(getTheme);
 
     if (text) {
         return (
             <div
                 {...attrs}
+                style={{
+                    background: tinycolor.mix(centerChannelBg, centerChannelColor, 8).toRgbString(),
```

Comment:
nit: it seems unusual to compute a colour based on our constants instead of using a constant defined by the theme directly. Are we sure that these colours will always "mix" correctly?

Reply:
This was a tricky one. It's a transparent bg, but there's an overlapping object so for this one element it needed to be opaque... testing in the 4 default themes we have yielded the expected colors, but there may be edge cases.

I'll mark it down for a more thorough UI check during UX review with custom themes.